### ISSUE TITLE
 Correct translation of urban block into Spanish

### ISSUE THREAD

**Issue by BikeRoad** [osmandapp/Osmand#8673]
Hola.
 
keytag place=city_block, the translation of city block in Spanish should be: "Manzana Urbana" or just "Manzana"and not "Distrito" (district)
![Screenshot_20200313-123245](https://user-images.githubusercontent.com/19896035/76672622-1d429e80-656c-11ea-8370-5e7ecf167116.png)

**Comment by vshcherb** [comment]
Unfortunately we merge it with "place=quarter" tag in our data probably you can adjust translation
```
<string name="poi_quarter">Quarter</string>
```

**Comment by BikeRoad** [comment]
Thanks vshcherb for responding. I would not know how to do it because I am not a programmer. I am a contributing user of OSM.

**Comment by scaidermern** [comment]
> Thanks vshcherb for responding. I would not know how to do it because I am not a programmer. I am a contributing user of OSM.

You can fix the translation at https://hosted.weblate.org/projects/osmand/. No programming knowledge is needed. Just search for the wrong translation: https://hosted.weblate.org/search/osmand/?q=Distrito.

**Comment by comradekingu** [comment]
@BikeRoad I changed city_type_district https://hosted.weblate.org/translate/osmand/main/es/?checksum=6f928fec668ceeb8 to "Manzana urbana" and poi_district https://hosted.weblate.org/translate/osmand/phrases/es/?checksum=92281d5b1e4546bf is still "distrito".

**Comment by franco999** [comment]
@vshcherb Does the string "quarter" refer to "city_block"? I don't understand why the tags are not separated, as they don't seem to mean the same thing.
https://wiki.openstreetmap.org/wiki/Tag:place%3Ddistrict
https://wiki.openstreetmap.org/wiki/Tag:place%3Dquarter
https://wiki.openstreetmap.org/wiki/Tag:place%3Dcity_block

@comradekingu What is the difference between the two strings? `city_type_district` and `poi_district`

**Comment by comradekingu** [comment]
@franco999 City block is just one "neighborhood" or so, district is bigger.

**Comment by franco999** [comment]
Correct me if I am wrong, but I understand that the order would be:
> .. city/town > suburb > quarter > neighborhood > city_block > plot

Where does "district" fit in that order?

**Comment by BikeRoad** [comment]
I agree with @franco999 in the order of values it exposes. What I see is that the key "quarter" and "city_block" are treated as if they are the same when they are not. Each key must have its translation separately. I suggest like this (In Spanish): 
City/town = "Ciudad"
Suburb = "Suburbio"
Quarter = ??? - I don't know what it could be.
Neighborhood = "Barrio"
City_block = "Manzana Urbana" or "Manzana"
Plot = "Parcela" or "Lote"
